packages/big-design/src/components/Worksheet/spec.tsx

```diff
@@ -866,4 +923,91 @@ describe('ModalEditor', () => {
 
     await waitForElement(() => screen.getAllByRole('combobox'));
   });
+
+  test('renders in disabled state', async () => {
+    render(<Worksheet columns={disabledColumns} items={items} onChange={handleChange} />);
+
+    const buttons = screen.queryAllByText('Edit');
+
+    expect(buttons).toStrictEqual([]);
+
+    await waitForElement(() => screen.getAllByRole('combobox'));
+  });
+});
+
+describe('disable', () => {
+  test('navigation works on disabled cells', async () => {
+    render(<Worksheet columns={disabledColumns} items={items} onChange={handleChange} />);
+
+    let cell = screen.getByText('Shoes Name Three');
```

Comment:
If all these are cell values you can grab them by `getByRole('cell', ...)`

Same with the ones below

Reply:
This one is a little misleading because it's actually fetching `p` elements. `getByRole` wouldn't work.